.gitignore

```diff
@@ -145,3 +145,12 @@ match_results.tab
 results.tab
 test.tab
 
+# VS Code
+.vscode
+
+# pytest_cache
+.pytest_cache/*
+.pytest_cache
+
+# Jupyter notebooks
+.ipynb
```

Comment:
We don't want to ignore Jupyter Notebooks. There may be cases where they are useful to include in the repo such as performing an interesting analysis with biohansel or showing how we can create a scheme for a particular organism.

Reply:
Addressed in commit 4277dffbf5dc652244a1425057a3cee485b4b4e1